models/articles-models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+function selectArticleById(articleId) {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = ${articleId}`)
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "not found" });
+      }
+      return result;
+    });
```

Comment:
You can return `result.rows[0]` here to ensure you send just the object, rather than the array